lms/djangoapps/courseware/user_state_client.py

```diff
@@ -250,6 +252,13 @@ def delete_many(self, username, block_keys, scope=Scope.user_state, fields=None)
         if scope != Scope.user_state:
             raise ValueError("Only Scope.user_state is supported")
 
+        if fields is None:
+            self._ddog_increment(evt_time, 'delete_many.empty_state')
+        else:
+            self._ddog_histogram(evt_time, 'delete_many.field_count', len(fields))
+
+        self._ddog_histogram(evt_time, 'delete_many.block_count', len(block_keys))
```

Comment:
`block_keys` has no `None` protection. Should the API call itself protect against this possibility?